lightly/api/api_workflow_selection.py

```diff
@@ -33,7 +33,17 @@ def _parse_active_learning_scores(scores: Union[np.ndarray, List]):
 
 
 class _SelectionMixin:
-    def upload_scores(self, al_scores: Dict[str, np.ndarray], query_tag_id: str):
+    def upload_scores(
+        self, al_scores: Dict[str, np.ndarray], query_tag_id: str
```

Comment:
```suggestion
        self, al_scores: Dict[str, NDArray[np.float_]], query_tag_id: str
```

You can import `NDArray` like this:
```
from numpy.typing import NDArray
```